Assets/_Scripts/MagicSpell/MovableUnit.cs

```diff
@@ -57,7 +58,7 @@ public class MovableUnit : MonoBehaviour {
 			return;
 		}
 		if ((other.gameObject.tag == TagList.Player
-		    || other.gameObject.tag == TagList.Fireball) && isMoving){
+		    || (other.gameObject.tag == TagList.Fireball && !bigger)) && isMoving){
```

Comment:
This part is too convoluted now.
You may refractor this part or leave a comment here at least.